### PR TITLE
feat: open editor on Edit and Reply on mobiles

### PR DESCRIPTION
**Pull request by evgeniy-ac**
#693

### CODE REVIEW

**Review by FrolYo — APPROVED** [review]

File: src/dotnet/UI.Blazor/Components/Menu/MenuEntry.razor [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <li @onclick="OnClick"
     class="ac-menu-item @Class"
     data-tooltip="@Tooltip"
+    data-editor-trigger="@(TriggerEditor ? "true" : "false")"
```
Comment (FrolYo) [inline src/dotnet/UI.Blazor/Components/Menu/MenuEntry.razor:6]: ToString?

File: src/dotnet/Chat.UI.Blazor/Components/MarkupEditor/markup-editor.ts [hunk 2]
```diff
@@ -653,3 +667,19 @@ function cleanPastedText(text: string) : string {
 function normalize(text: string) : string {
     return text.normalize().replace(CrlfRe, "\n");
 }
+
+function focusAndOpenKeyboard(el: HTMLDivElement, timeout: number) {
+    const tempElement = document.createElement('input');
```
Comment (FrolYo) [inline src/dotnet/Chat.UI.Blazor/Components/MarkupEditor/markup-editor.ts:672]: afair there is editable div in our markup editor, so it should focusable by default, shouldn't it?

Comment (evgeniy-ac) [reply src/dotnet/Chat.UI.Blazor/Components/MarkupEditor/markup-editor.ts:672]: it's a hack for iOS

File: src/dotnet/UI.Blazor/Components/Menu/MenuEntry.razor (hunk 1 above)
Comment (evgeniy-ac) [reply src/dotnet/UI.Blazor/Components/Menu/MenuEntry.razor]: Done

**Pull request merged**